docs/index.md

```diff
@@ -101,6 +101,21 @@ way. to use the Trino connector, please see:
 * [How to use Gravitino Trino connector](./trino-connector/index.md): a complete guide to use Gravitino
   Trino connector.
 
+### Server administration
+
+Gravitino provides several ways to configure and manage the Gravitino server. Please see:
+
+* [Security](./security.md): provides the security configurations for Gravitino, including HTTPS
```

Comment:
configuration

Reply:
There's not one security configuration. So using plural and delete "the" may be correct?

Reply:
There's not one security configuration. So using plural and delete "the" may be correct?

Reply:
deleting the would be better, but it's a minor issue